tests/app/api/test_reporting.py

```diff
@@ -22,34 +22,12 @@ def test_reporting_details_eq_success(self):
                                                                             '/14fb3e68-4dca-46db-bf49-04b84e07e999',
                           json=self.reporting_response, status=200)
 
-            report = reporting_details('eq',
-                                       '57586798-74e3-49fd-93da-a782ec5f5129',
+            report = reporting_details('57586798-74e3-49fd-93da-a782ec5f5129',
                                        '14fb3e68-4dca-46db-bf49-04b84e07e999')
             decoded_report = json.loads(report)
 
         self.assertEqual(decoded_report, self.reporting_response)
 
-    @responses.activate
-    def test_reporting_details_seft_success(self):
-        with self.app.app_context():
-            seft_reporting_response = {'metadata': {'collectionExerciseId': '14fb3e68-4dca-46db-bf49-04b84e07e999',
-                                                    'timeUpdated': 1533895381.534031},
-                                       'report': {'downloads': 259, 'accountsPending': 402, 'accountsEnrolled': 259,
-                                                  'notStarted': 457, 'uploads': 160, 'sampleSize': 716}}
-            responses.add(responses.GET, self.app.config['REPORTING_URL'] + '/reporting-api/v1/response-dashboard'
-                                                                            '/survey'
-                                                                            '/57586798-74e3-49fd-93da-a782ec5f5129'
-                                                                            '/collection-exercise'
-                                                                            '/14fb3e68-4dca-46db-bf49-04b84e07e999',
-                          json=self.reporting_response, status=200)
-
-            report = reporting_details('seft',
-                                       '57586798-74e3-49fd-93da-a782ec5f5129',
-                                       '14fb3e68-4dca-46db-bf49-04b84e07e999')
-            decoded_report = json.loads(report)
-
-        self.assertEqual(decoded_report, seft_reporting_response)
-
     def test_reporting_details_malformed_collex_id(self):
         response = self.test_client.get('/dashboard/reporting/eq'
```

Comment:
Collection instrument type is still on this test. It's not hitting anything in the repo